docs/tutorials/02-registries-repositories.md

```diff
@@ -0,0 +1,133 @@
+# Buildah Tutorial 2
+## Using Buildah with container registries
+
+The purpose of this tutorial is to demonstrate how Buildah can be used to move OCI compliant images in and out of private or public registries. 
+
+In the [first tutorial](https://github.com/projectatomic/buildah/blob/master/docs/tutorials/01-intro.md) we built an image from scratch that we called `fedora-bashecho` and we pushed it to a local docker repository using the `docker-daemon` protocol. We are going to use the same image to push to a private docker registry.
+
+First we must pull down a registry using:
+
+    # registry=$(buildah from registry)
+
+It is worth pointing out that the `from` command can also use other protocols beyond the default (and implicity assumed) order that first looks in local containers-storage (containers-storage:) and then looks in the Docker hub (docker:). So, for example, if you already had a registry container image in a local docker registry then you could use the following:
```

Comment:
Docker registry.  Gotta hand out the big 'D'.

Reply:
I used it on purpose here. I.e. this is the docker project registry and not the Docker Inc. registry. I use big D for that when I mention the hub. 

Do I have it wrong?

Reply:
Done